packages/eslint-config/README.md

```diff
@@ -0,0 +1,5 @@
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
+
+# @sap-cloud-sdk/eslint-config
+
+This package contains a shareable version of our `eslint` configuration.
```

Comment:
Maybe you can give a bit more context - who is our. Imagine you would find this on npmjs.com.